index.html

```diff
@@ -605,7 +605,224 @@ <h2>Assertion Method</h2>
+    <p>
+      In contrast to this, the JSON representation (with media type `application/did+json`) does not have this semantic
+      capability; instead, (non-LD) JSON documents always need agreement on semantics in some out-of-band way that is
+      not part of the document itself. In the case of DID documents using the JSON representation, those semantics are
+      established by the <a href="https://w3c.github.io/did-spec-registries/">DID Spec Registries</a>. Therefore,
+      establishing semantics of DID document properties in the JSON representation does not require any
+      “representation-specific entries”, as is the case with the JSON-LD representation. The same is true for the CBOR
+      representation.
+    </p>
+
+    <p>
+      Other examples of “representation-specific entries” could be XML namespace definitions, or YAML tags, although
+      these particular representations have not been defined at the time of this writing.
+    </p>
+
+    <p>
+      If an entry that is specific to one representation appears in another representation, it can be called a
+      "representation-foreign entry". An example of this is an `@context` entry appearing in the `application/did+json`
+      representation.
+    </p>
+
+    <p>
+      Examples of “representation-specific entries” and “representation-foreign entries”:
+    </p>
+
+    <figure id="representation-entries">
+      <img style="margin: auto; display: block;"
+           src="diagrams/representation-specific-and-foreign-entries.png" alt="
+Examples of representation-specific and representation-foreign entries in various representations.">
+      <figcaption>
+        Representation-specific and representation-foreign entries.
+      </figcaption>
+    </figure>
+
+    <section class="informative">
+      <h3>
+        Avoid representation-foreign entries
+      </h3>
+
+      <p class="advisement">
+        It is bad practice and potentially harmful for producers to include "representation-foreign entries" in DID
+        documents.
+      </p>
+
+      <p>
+        Different implementations of DID document consumers may or may not be able to properly understand the meaning
+        and processing rules of representation-foreign entries. Since DIDs and DID documents form an open ecosystem with
+        many producers and consumers, a producer cannot anticipate the behavior of a consumer with regard to
+        representation-foreign entries.
+      </p>
+
+      <p>
+        For example, if a producer adds an `@context` entry to a DID document in the `application/did+json`
+        representation, some consumers (those that understand JSON-LD, such as JSON-LD document loaders) may process it
+        in one way, while other consumers (those that only understand plain JSON, without any JSON-LD specific rules)
+        may process it in a different way.
+      </p>
+
+      <p>
+        This can lead to inconsistent behavior by consumers that cannot be predicted by a producer.
```

Comment:
Consumers can do anything, anyway.  Producers cannot predict all possible consumer behaviors.  This has nothing to do with whether `@context` is included in or excluded from JSON docs.

Reply:
It is important that DID documents convey the same semantics for all registered properties, across all representations and all conforming consumers. Including `@context` in the non-LD JSON representation threatens this objective, since different consumers may interpret properties in the DID document differently.